pool/app/utils/database.ml

```diff
@@ -20,18 +21,36 @@ module Dynparam = struct
   let add t x (Pack (t', x')) = Pack (Caqti_type.t2 t' t, (x', x))
 end
 
-let raise_caqti_error ?tags =
+let raise_caqti_error ?req ?tags =
```

Comment:
Is the request argument optional because of the transactions? Would exceptions in the transactions be reported?

Reply:
Yes, the transactions are getting reported by the single request executions.